lib/oelite/parse/oeparse.py

```diff
@@ -582,13 +582,13 @@ def inherit(self, filename, p):
             return
         if not os.path.isabs(filename) and not filename.endswith(".oeclass"):
             filename = os.path.join("classes", "%s.oeclass"%(filename))
-        if not "__inherits" in self.meta:
-            self.meta["__inherits"] = [filename]
-        else:
-            __inherits = self.meta["__inherits"]
+        __inherits = self.meta.get("__inherits")
+        if __inherits:
```

Comment:
```
ravi@prevas-ravi:/mnt/xfs/devel/oe-lite/meta/core$ git grep setdefault
ravi@prevas-ravi:/mnt/xfs/devel/oe-lite/meta/core$ 
```


I suppose I know what you mean, but I don't think DictMeta has a method for 'get the value if it exists, otherwise set it to this and return that'... but I don't really care either way, so feel free to omit this patch and/or replace with something else